compose.yml

```diff
@@ -222,6 +222,14 @@ services:
             - RESTIC_STREAM=yes
             - RESTIC_STREAM_CMD=tar -cf - -C /data_mediawiki/ /data/database.sql.gz images
             - RESTIC_STREAM_FILENAME=full.images.database.tar
+            - RESTIC_REPOSITORY_FILE=/run/secrets/restic_s3_repository
+            - RESTIC_PASSWORD_FILE=/run/secrets/restic_password
+            - AWS_SHARED_CREDENTIALS_FILE=/run/secrets/aws_credentials
```

Comment:
I don't think we use `AWS_SHARED_CREDENTIALS_FILE`, we use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` instead

Reply:
This is an exception, there is only one way is to use `AWS_SHARED_CREDENTIALS_FILE`

Reply:
Right, we probably eventually should add _FILE support to the Restic image for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`